src/viper/syntax.ml

```diff
@@ -72,6 +72,8 @@ and stmt' =
   | IfS of exp * seqn * seqn
   | WhileS of exp * invariants * seqn
   | LabelS of id * invariants
+  | PreconditionS of exp
```

Comment:
These are currently get translated as `stmt` lists, but before constructing the `MethodI` they get converted to the pre/post entries of `MethodI`, and stripped from the body. 63189ba6b5b0d32fa95b3edfad89ff5eaad66881